pkg/config/config.go

```diff
@@ -213,12 +221,8 @@ func ParseCliArgs() (config Config, err error) {
 		config.PodTerminationGracePeriod = gracePeriod
 	}
 
-	switch strings.ToLower(config.LogLevel) {
-	case "info":
-	case "debug":
-	case "error":
-	default:
-		return config, fmt.Errorf("Invalid log-level passed: %s  Should be one of: info, debug, error", config.LogLevel)
+	if err := validateLogLevel(strings.ToLower(config.LogLevel)); err != nil {
```

Comment:
I don't think this adds value as a separate function, I'd prefer you revert this

Reply:
It doesn't add much value, I just did it to reduce the cyclomatic complexity, otherwise `go-report-card-test` would fail.

Reply:
I say revert it and don't worry about the report card test, we'll take care of that afterwards

Reply:
Done.